python/hail/methods/impex.py

```diff
@@ -629,8 +630,124 @@ def get_vcf_metadata(path):
                drop_samples=False, call_fields=[], reference_genome=None, contig_recoding=None):
-    return Env.hc().import_vcf(path, force, force_bgz, header_file, min_partitions,
-                               drop_samples, call_fields, reference_genome, contig_recoding).to_hail2()
+    """Import VCF file(s) as a matrix table.
+
+    Examples
+    --------
+
+    >>> ds = methods.import_vcf('data/example2.vcf.bgz')
+
+    Notes
+    -----
+
+    Hail is designed to be maximally compatible with files in the `VCF v4.2
+    spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`__.
+
+    :func:`.import_vcf` takes a list of VCF files to load. All files must have
+    the same header and the same set of samples in the same order (e.g., a
+    dataset split by chromosome). Files can be specified as :ref:`Hadoop glob
+    patterns <sec-hadoop-glob>`.
+
+
+    Ensure that the VCF file is correctly prepared for import: VCFs should
+    either be uncompressed (*.vcf*) or block compressed (*.vcf.bgz*). If you
+    have a large compressed VCF that ends in *.vcf.gz*, it is likely that the
+    file is actually block-compressed, and you should rename the file to
+    ".vcf.bgz" accordingly. If you actually have a standard gzipped file, it is
+    possible to import it to Hail using the ``force`` optional parameter.
+    However, this is not recommended -- all parsing will have to take place on
+    one node because gzip decompression is not parallelizable. In this case,
+    import will take significantly longer.
+
+
+    :func:`.import_vcf` does not perform deduplication - if the provided VCF(s)
+    contain multiple records with the same chrom, pos, ref, alt, all these
+    records will be imported as-is and will not be collapsed into a single
+    variant.
+
+    .. note::
+
+        Using the **FILTER** field:
+
+        The information in the FILTER field of a VCF is contained in the
+        ``filters`` row field. This annotation is a ``Set[String]`` and can be
+        queried for filter membership with expressions like
+        ``ds.filters.contains("VQSRTranche99.5...")``. Variants that are flagged
+        as "PASS" will have no filters applied; for these variants,
+        ``ds.filters.is_empty()`` is ``True``. Thus, filtering to PASS variants
+        can be done with :meth:`.MatrixTable.filter_rows` as follows:
+
+
+        >>> pass_ds = dataset.filter_rows(ds.filters.is_empty())
+
+
+    **Column Fields**
+
+    - **s** (*String*) -- Column key. This is the sample ID.
+
+    **Row Fields**
+
+    - **v** (*Variant*) -- Row key. This is the variant created from the CHROM,
+      POS, REF, and ALT fields.
+    - **filters** (*Set[String]*) -- Set containing all filters applied to a
+      variant.
+    - **rsid** (*String*) -- rsID of the variant.
+    - **qual** (*Double*) -- Floating-point number in the QUAL field.
+    - **info** (*Struct*) -- All INFO fields defined in the VCF header
```

Comment:
You use double backticks for types above but stars here.  I guess we should be consistent?

Reply:
We don't really follow a convention elsewhere, I guess here's an opportunity to set it. I think we should use italics (single backticks) for field names, and link to the Type class. I'l change that here.

Reply:
also, oops -- left "Double"!